src/app/clusters/ota-provider/ota-provider.h

```diff
@@ -27,7 +27,8 @@ namespace OTAProvider {
 
 void SetDelegate(chip::EndpointId endpointId, OTAProviderDelegate * delegate);
 
-}
+void MatterOTAProviderPluginServerInitCallback();
+} // namespace OTAProvider
```

Comment:
No, the MatterOTAProviderPluginServerInitCallback function needs to be outside all namespaces to match the normal API for this.

Reply:
removed.